src/api/searchApi.ts

```diff
@@ -159,7 +159,8 @@ export const fetchImportCandidates = async ({
     params.set(ImportCandidatesSearchParam.Query, query);
   }
   if (publicationYear) {
-    params.set(ImportCandidatesSearchParam.PublicationYear, publicationYear.toString());
+    const yearString = publicationYear.toString();
+    params.set(ImportCandidatesSearchParam.PublicationYear, `${yearString},${yearString}`);
```

Comment:
I de siste søke-api-endringene så godtar man et enkelt årstall for å søke på et gitt år. Funker det på denne også?

```suggestion
    params.set(ImportCandidatesSearchParam.PublicationYear, yearString);
```

Reply:
Har en dialog med Stig om det: https://github.com/BIBSYSDEV/NVA-Frontend/pull/5590#discussion_r1590942474
Ikke på plass atm, men håper å kunne ta vekk den der snart, feks når jeg legger inn aggregeringer

Reply:
Se der ja, scrollet kanskje litt for raskt forbi de uekspanderte resolved-kommentarene 😇

Reply:
forventer ikke at folk leser resolved comments nei, bruker det mest som en huskeliste for meg selv før jeg setter den til ready for review. Så det er en good spot! :D